internal/service/hadoop/hadoop.go

```diff
@@ -238,38 +189,104 @@ func (h *hadoopResource) Schema(ctx context.Context, req resource.SchemaRequest,
 					),
 				},
 			},
+			"image_product_code": schema.StringAttribute{
+				Optional: true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+					stringplanmodifier.RequiresReplace(),
+				},
+				Description: "default: latest version",
+			},
+			"edge_node_product_code": schema.StringAttribute{
+				Optional: true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+				Description: "default: minimum spec",
+			},
+			"master_node_product_code": schema.StringAttribute{
+				Optional: true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+				Description: "default: minimum spec",
+			},
+			"worker_node_product_code": schema.StringAttribute{
+				Optional: true,
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+				Description: "default: minimum spec",
+			},
+			"add_on_code_list": schema.ListAttribute{
+				ElementType: types.StringType,
+				Optional:    true,
+				PlanModifiers: []planmodifier.List{
+					listplanmodifier.RequiresReplace(),
+				},
+				Description: "this attribute can used over 1.5 version",
+			},
 			"worker_node_count": schema.Int64Attribute{
 				Optional: true,
+				Computed: true,
+				Default:  int64default.StaticInt64(2),
+				PlanModifiers: []planmodifier.Int64{
+					int64planmodifier.UseStateForUnknown(),
+				},
 				Validators: []validator.Int64{
-					int64validator.Between(2, 8),
+					int64validator.AtLeast(2),
 				},
 				Description: "default: 2",
 			},
 			"use_kdc": schema.BoolAttribute{
 				Optional: true,
+				Computed: true,
+				Default:  booldefault.StaticBool(false),
 				PlanModifiers: []planmodifier.Bool{
 					boolplanmodifier.RequiresReplace(),
 				},
 				Description: "default: false",
```

Comment:
(trivial) If the default value has been presented in the schema (Default: xxx), self-explanation about the default value in the description seems unnecessary.